src/test/javascript/spec/component/complaints/complaint-response.service.spec.ts

```diff
@@ -67,6 +67,12 @@ describe('ComplaintResponseService', () => {
         expect(isLocked).toBeTrue();
     });
 
+    it('should correctly calculate that complaint response is locked by other user', () => {
+        const lockedComplaintResponse = setupLockTest('test', false, 'test2', false);
+        const isLocked = complaintResponseService.isComplaintResponseLockedForLoggedInUser(lockedComplaintResponse, new TextExercise(undefined, undefined));
+        expect(isLocked).toBeFalse();
```

Comment:
double check that this test actually fails on `develop` and that it covers enough `expect` statements to verify that the functionality is really implemented correctly

Reply:
This actually fails 👍